cypress/support/commands.js

```diff
@@ -10,7 +10,27 @@
 //
 //
 // -- This is a parent command --
-// Cypress.Commands.add("login", (email, password) => { ... })
+Cypress.Commands.add("login", (email, password) => {
+
+  // it is ok for the email to be visible in the Command Log
+  expect(email, "email was set").to.be.a("string").and.not.be.empty;
+  // but the password value should not be shown
+  if (typeof password !== "string" || !password) {
+    throw new Error("Missing password value, set using CYPRESS_password= or in cypress env file...");
+  }
+
+  cy.get('input[name="emaial"]').type(email);
+
+  cy.get("[name=passwd]")
+    .type(password, { log: false })
```

Comment:
Not sure why we're cleaning up the log messages here, can we remove it if this is a remnant of some test?

Reply:
This is so the cypress log window does not show the password when running tests.